models/gs2/README.md

```diff
@@ -0,0 +1,48 @@
+# GS2 Fusion Plasma Simulation
+
+## Overview
+This model uses [GS2](https://gyrokinetics.gitlab.io/gs2/index.html) to simulate plasma in a spherical tokamak. It aims to study various modes in the plasma due to microinstabilities. The current setup terminates once an unstable mode is found. Otherwise, it will continue until reaching a fixed timestep. Therefore, the runtime varies depending on the input parameters.
+
+## Authors
+- [Chung Ming Loi](mailto:chung.m.loi@durham.ac.uk)
+
+
+## Run
+```
+docker run -it -p 4242:4242 linusseelinger/model-achlys:latest
```

Comment:
I don't think this is the right Docker image. This looks like the one for Tritium desorption.